.gitignore

```diff
@@ -79,3 +79,6 @@ data
 
 # PyCharm
 .idea
+
+# bash scripts
+/scripts
```

Comment:
perhaps we can avoid committing this but it's only in the .gitignore

Reply:
As it is a personnal thing I don't think it belongs in the public repository

Reply:
Use https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer or https://help.github.com/en/github/using-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file

Reply:
I erased that, but thx it's good to know